add-to-pydotorg.py

```diff
@@ -244,14 +244,17 @@ def has_sigstore_signature(rfile):
     print('Signging release files with Sigstore')
     run_cmd(['python3', '-m', 'sigstore', 'sign', '--oidc-disable-ambient-providers'] + filenames)
 
+    # Update permissions on Sigstore verification materials to match other downloads
+    run_cmd(["find", ftp_root, "-type", "f", "\(", "-iname", "\*.sig", "-o", "-iname", "\*.crt", "\)", "exec", "chmod", "644", "{}", "\\;"])
```

Comment:
I reverted this in 21e1adaf9b04ffbcf8d3699bf11f1ecac3db239c because I'm not okay with touching all files in ftp_root. Ideally `sigstore sign` should produce properly accessible .sig and .crt files. But in the absence of this, chmodding *only* the files from `unsigned_files` (+ .sig/.crt extensions) would be okay.

Reply:
I made this suggested change in 6846215910b2bd6db82f21fd68289afe8c5b8f76.